packages/cli/test/unit/commands/teams/add.test.ts

```diff
@@ -11,22 +11,39 @@ describe('teams add', () => {
   beforeEach(() => {
     useUser();
     useTeams(team.id);
-  });
 
-  beforeEach(() => {
     client.config = {
       currentTeam: currentTeamId,
     };
   });
 
-  client.scenario.post(`/teams`, (req, res) => {
-    return res.json(team);
-  });
```

Comment:
The scenario are not needed for `--help`. Moved them into the one other test in this file for now.